pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -17,11 +17,6 @@ install -m 755 "${LIMA_CIDATA_MNT}"/lima-guestagent /usr/local/bin/lima-guestage
 
```

Comment:
We probably need to uninstall the previous version of the guest agent

Reply:
Done

Reply:
> We probably need to uninstall the previous version of the guest agent

Just out of curiosity: why do we need to remove it? `install` should be able to overwrite the file. What am I missing?

Reply:
Not overwritten because we switched away from non-root to root

Reply:
I meant we have to remove non-root systemd unit

Reply:
> I meant we have to remove non-root systemd unit

Ok, thanks, because that is not what https://github.com/mattfarina/lima/commit/3aba8f39fcccce1102e963db4c0ae34809db02e2 is doing...

Reply:
This makes sense. Just pushed a change to remove the legacy systemd configuration.